cross-chain/arbitrum/package.json

```
@@ -0,0 +1,76 @@
+{
+  "name": "cross-chain-arbitrum",
```

According to RFC 8, the package name should be:
```suggestion
  "name": "@keep-network/tbtc-v2-arbitrum",
```

---

Done [d7de7a3](https://github.com/keep-network/tbtc-v2/pull/554/commits/d7de7a311c53cf8d9b68e861621c7dc6daa981d4)